pages/categoryPage/category.html

```diff
@@ -2,10 +2,478 @@
+                            <p>Long Pants & Skirts</p>
+                            <span>(9)</span>
+                        </div>
+                        <div class="subListElement">
+                            <p>Jeans & Denims</p>
+                            <span>(10)</span>
+                        </div>
+                    </div>
+                </li>
+
+            </ul>
+        </div>
+
+        <div class="filterBlock">
+            <p>COLOR</p>
+            <div class="colorFilters">
+                <div class="colorRow">
+                    <input type="checkbox" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #fff;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #000;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #51a909;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #497bee;" class="colorCheckbox"/>
+                </div>
+                <div class="colorRow">
+                    <input type="checkbox" style="background-color: #fee479;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #f7931d;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #7b3a94;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #dd1cdf;" class="colorCheckbox"/>
+                    <input type="checkbox" style="background-color: #cc0d0d;" class="colorCheckbox"/>
+                </div>
+            </div>
+        </div>
+
+        <div class="filterBlock">
+            <p>BRANDS</p>
+            <div class="brandFilters">
+                <label>
+                    <input type="checkbox" class="checkMarkCheckbox">
+                    <span>Calvin Klein</span>
+                </label>
+                <label>
+                    <input type="checkbox" class="checkMarkCheckbox">
+                    <span>Zara</span>
+                </label>
+                <label>
+                    <input type="checkbox" class="checkMarkCheckbox">
+                    <span>Adidas</span>
+                </label>
+                <label>
+                    <input type="checkbox" class="checkMarkCheckbox">
+                    <span>Nike</span>
+                </label>
+                <label>
+                    <input type="checkbox" class="checkMarkCheckbox">
+                    <span>Mango</span>
+                </label>
+            </div>
+        </div>
+
+        <div class="filterBlock">
+            <p>PRICE</p>
+            <div class="priceFilter">
+                <div class="priceSlider">
+                    <div class="priceProgress"></div>
+                </div>
+                <div class="rangeInput">
+                    <input type="range" class="rangeMin" min="0" max="500" value="0" step="10">
```

Comment:
to do: get min and max price from products json and use this in your js file

Reply:
Also, get color from products json and use this in your js file